src/Omega_h_simModel2d.cpp

```diff
@@ -0,0 +1,466 @@
+#include <SimModel.h>
+#include <SimUtil.h>
+#include "Omega_h_model2d.hpp"
+#include "Omega_h_profile.hpp"
+#include "Omega_h_map.hpp"  // get_degrees
+#include "Omega_h_adj.hpp"  // invert_adj
+#include "Omega_h_array_ops.hpp" // operator==(LOs,LOs)
+#include <map>
+#include <algorithm> //std::fill
+#include <numeric> //std::exclusive_scan
+
+namespace Omega_h {
+
```

Comment:
Pretty much every function in this file should be marked [[nodiscard]] and static since you will not be using it in other translation units.